src/eXpansion/Framework/Gui/Layouts/LayoutRow.php

```diff
@@ -197,7 +195,7 @@ public function setHeight($height)
 
     /**
      * @param float $width
-     * @return LayoutRow
+     * @return layoutRow
```

Comment:
why are they small again ?

Reply:
i had to revert the file since the regression errors... there's still regression errors

Reply:
since the whole ui broke... negative margins and setting wrongly the frame size... etc.. etc..